contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -89,12 +90,6 @@ contract TrueCreditAgency is UpgradeableClaimable, ITrueCreditAgency {
     /// @dev next payment due time per borrower for each pool
     mapping(ITrueFiPool2 => mapping(address => uint256)) public nextInterestRepayTime;
 
-    /// @dev whitelist for allowing pools to have lines of credit
-    mapping(ITrueFiPool2 => bool) public isPoolAllowed;
-
-    /// @dev array of pools with lines of credit
-    ITrueFiPool2[] public pools;
```

Comment:
Didn't we plan to deploy older audited version first? This will break the storage

Reply:
Thanks for catching this. I believe the team would like to launch the audited version first and then follow up later with this version.

I'll revert the deletions here and mark these as DEPRECATED.